File: Cointln/cointl.cs
// conintl.cs - Localización Nativa en .NET 10
var builder = WebApplication.CreateBuilder(args);
var app = builder.Build();

app.MapGet("/", (HttpContext context) => {
    string n = context.Request.Query["n"] ?? "0";
    
    // Simulación de algoritmo local en C# sin llamar a servicios externos SOAP
    return n == "10" ? "diez" : "Procesado localmente con el motor nativo de C#";
});

app.Run("http://localhost:8003");

File: Cointln/cointl.go
// conintl.go - Localización Nativa en Go
package main

import (
	"fmt"
	"net/http"
)

func main() {
	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
		numero := r.URL.Query().Get("n")
		
		// Conversión nativa directa
		if numero == "10" {
			fmt.Fprint(w, "diez")
		} else {
			fmt.Fprint(w, "Procesado mediante algoritmo nativo en Go")
		}
	})

	fmt.Println("Servidor conintl (Go) activo en el puerto 8003...")
	http.ListenAndServe(":8003", nil)
}